### ISSUE TITLE
Not getting any reponse from onFriendRequest()

### ISSUE THREAD

**Issue by MS-Jahan** [carpedm20/fbchat#506]
## Description of the problem
Not getting any reponse from onFriendRequest().

## Code I tried

```
def onFriendRequest(self, from_id, msg, **kwargs):
        user = self.fetchUserInfo(from_id)[from_id]
        content = "Friend request from {}".format(user.name)
        print(content)
```

Is it a bug or Am I doing it wrong?

**Comment by madsmtm** [comment]
Hmm, I haven't been able to test this, so it's very probable that this doesn't work since #494. Try enabling debug output like this, and give me the data (after you've removed sensitive information).

```python
# before your application code
import logging
logging.basicConfig(level=logging.DEBUG)
```

**Comment by madsmtm** [comment]
Just got a friend to help me test it, and I've fixed the issue in 45303005b8b635fd27374fa0ba3600d7a1b7c3bb, and released `v1.9.4`. Thanks for the report 👍
